child_server/child_game/cell.py

```diff
@@ -115,15 +129,14 @@ def render(self, **keyword_parameters):
                                 'Player': 'a',
                                 'SentryTurret': 'b',
                                 'SpikeTrap': 'b',
-                                'Fence': 'e',
+                                'Fence': 'c',
                                 'Pharmacy': 'b',
                                 'Hospital': 'b',
                                 'Door': 'b',
                                 'RespawnBeacon': 'b',
                                 'OreGenerator': 'c',
                                 'OreDeposit': 'd',
-                                'Loot': 'd',
-                                'Fence': 'd'
```

Comment:
There having been two entries for Fence was really silly. https://github.com/baxter-cs/AndOre/commit/0ddee36192cfb65db4f4362cd76eb80c74145332